### DISCUSSION TITLE
Container_name.cursor = "pointer" not working after pixi-legacy7. 0.4.

### DISCUSSION THREAD

**Discussion by ajith-1998** [pixijs/pixijs#8995] [Q&A]
Before pixi v7 buttonMode properties are working properly for all displayobjects. 

After pixi v7 update it won't working, so, I've changed those into objectname.cursor = "pointer".

Sprite, Text, Animatedsprite's cursor properties are working fine, but the containers cursor property is not working. Container interactions working properly like before. container children cursor properties also working fine.

**Comment by bigtimebuddy** [comment]
Can you provide a reproduction of 7.0.4 and 7.0.5?

**Reply by ajith-1998** [reply]
Hi,

I think you misunderstood my question. 
I have developed a game where i use sprite and text to be inside a container and i added a code containername.cursor = "pointer" instead of objectname.buttonmode = true which is deprecated in PIXIJS 7.0.4. Even i updated with PIXIJS 7.0.5 the result is same, no fix here too.

Now my issue is containername.cursor = "pointer" is not working fine. it doesn't have hand cursor in the output. Can you help me?

In other words, My problem is that hand cursor is not working for containers (whereas it works fine for texts and sprites) when i use pixijs 7.0.4 or 7.0.5. So when i try to fix this i researched thro forum and i had no answers. So i investigated the game engine, where i simply edited one code as a test but that's not a solution, just my observation, but what i saw is that it creates hand cursor for all.

 I just want you to simply fix the container properties to include hand cursor. Hope now i made it very clear this time. Kindly let me know about it when you fixed it.

**Reply by miltoncandelero** [reply]
What we mean is that we do not see the same problem you are.

It just works for us.
https://www.pixiplayground.com/#/edit/kG-gxxRoIXPjWuxUC1tAW
